### ISSUE TITLE
Test Lists Page Interactive Elements for Accessibility

### ISSUE THREAD

**Issue by bpmcneilly** [internetarchive/openlibrary#4901]
# Page to Test:

Lists - https://openlibrary.org/lists

# Steps

Steps for this task are outlined in #4897
 
# Expectations

Test the specified page according to the steps outlined in the parent issue. Create individual bugs with steps to reproduce for any found violations on this page. Reference those bugs on this tracking issue.

Once the page has been tested & all bugs are found, close this issue.

**Comment by Yashs911** [comment]
@bpmcneilly I would like to work on this.

**Comment by Yashs911** [comment]
No issues. Tested both http://staging.openlibrary.org/lists & https://openlibrary.org/lists
